packages/common/src/types/transition-log.ts

```diff
@@ -1,12 +1,18 @@
 import { TDbRecord } from './common';
 import { TMerkleProofSerialized } from './merkle-tree';
+import { EDocumentOperation } from './queue';
 
 export type TTransitionLog = {
   merkleRootNew: string;
   merkleProof: TMerkleProofSerialized[];
   leafOld: string;
   leafNew: string;
   operationNumber: bigint;
+  operationKind: EDocumentOperation;
+  // Should be null if the operation kind is create
+  previousDocumentObjectId: string | null;
```

Comment:
Should be `documentObjectIdPrevious` and `documentObjectIdCurrent`, it fits our naming convention